app/src/main/java/nu/yona/app/YonaApplication.java

```diff
@@ -99,6 +101,7 @@ public static Tracker getTracker() {
                 tracker.setAppVersion(pInfo.versionName + mContext.getString(R.string.space) + pInfo.versionCode);
                 tracker.send(new HitBuilders.ScreenViewBuilder().build());
             } catch (PackageManager.NameNotFoundException e) {
+                AppUtils.reportException(YonaApplication.class.getSimpleName(), e, Thread.currentThread(), null);
                 nu.yona.app.utils.Logger.loge(YonaApplication.class.getSimpleName(), e.getMessage());
```

Comment:
The above line should obsolete this one. ``reportException`` should log it as error if it doesn't already do so.

Reply:
Updated.